packages/nx/src/native/logger/mod.rs

```diff
@@ -88,13 +90,30 @@ where
 /// - `NX_NATIVE_LOGGING=nx=trace` - enable all logs for the `nx` (this) crate
 /// - `NX_NATIVE_LOGGING=nx::native::tasks::hashers::hash_project_files=trace` - enable all logs for the `hash_project_files` module
 /// - `NX_NATIVE_LOGGING=[{project_name=project}]` - enable logs that contain the project in its span
+/// NX_NATIVE_FILE_LOGGING acts the same but logs to .nx/workspace-data/nx.log instead of stdout
 pub(crate) fn enable_logger() {
-    let env_filter =
-        EnvFilter::try_from_env("NX_NATIVE_LOGGING").unwrap_or_else(|_| EnvFilter::new("ERROR"));
-    _ = tracing_subscriber::fmt()
-        .with_env_filter(env_filter)
+    let stdout_layer = tracing_subscriber::fmt::layer()
         .with_ansi(std::io::stdout().is_terminal())
+        .with_writer(std::io::stdout)
```

Comment:
I don't think this should have changed.

Reply:
I was out of the room when you guys reviewed this one so I'll leave it to you

Reply:
This one is fine as is `tracing` always used to write to `stdout`. This was added because we need to add a writer.